scss/_settings_colors.scss

```diff
@@ -42,7 +42,7 @@ $color-label-validated: #006b75;
 $color-code-background: rgba($color-x-dark, 0.03);
 $color-code-background-dark: rgba($color-x-light, 0.3);
 $color-code-heading-background: rgba($color-x-dark, 0.08);
-$color-navigation-highlight: #e95420;
+$color-navigation-highlight: #e95420 !default;
```

Comment:
Why is this not using one the themed colors? I know brand is dark by default by it feels like this should be accent color. The use of accent color is to add a small flourish of colour to the patterns.